docs/src/gallery/scales.md

```diff
@@ -97,6 +97,27 @@ p2 = plot(x=xs, y=ys, z=zs, Geom.contour, Scale.color_none);
 hstack(p1,p2)
 ```
 
+## [`Scale.linestyle_discrete`](@ref)
+
+```@example
+using DataFrames, Gadfly, RDatasets
+using StatsBase: winsor
+set_default_plot_size(18cm, 8cm)
```

Comment:
I'll probably expand it to 21cm after I implement `Guide.linekey`, since a linekey will appear for both plots. For now its 18cm to signify future growth!